src/core/api/option_parsers.ts

```diff
@@ -547,6 +552,16 @@ function parseLoadVideoOptions(
   const manifestUpdateUrl = options.transportOptions?.manifestUpdateUrl;
   const minimumManifestUpdateInterval =
     options.transportOptions?.minimumManifestUpdateInterval ?? 0;
+  let audioTrackSwitchingMode = options.transportOptions?.audioTrackSwitchingMode ??
```

Comment:
After testing, I just noticed that this option in the `transportOptions`. Why is that?

Reply:
Yea, you are right, `audioTrackSwitchingMode` has nothing to do here. I will do as `manualBitrateSwitchingMode`.